nntrainer/layers/layer_impl.cpp

```diff
@@ -16,6 +16,7 @@
 #include <vector>
 
 #include <common_properties.h>
+#include <layer_internal.h>
```

Comment:
Could you check line 7: base Optimizer Implementation Class??

Reply:
done.